update-format-crau/src/delta_update.rs

```diff
@@ -103,13 +103,13 @@ pub fn get_header_data_length(header: &DeltaUpdateFileHeader, manifest: &proto::
     // Payload data structure:
     //  | header | manifest | data blobs | signatures |
 
-    header.translate_offset(manifest.signatures_offset.unwrap()) as usize
+    header.translate_offset(manifest.signatures_offset.unwrap_or_default()) as usize
```

Comment:
I think `or_default` will lead to a later error when we try to use `data blobs` as signature data. I would rather error out early here.

Reply:
(Change the return type of the function to Result)

Reply:
Done